conanfile.py

```diff
@@ -278,15 +279,18 @@ def system_requirements(self):
                     installer.install(item + self._system_package_architecture())
 
     def source(self):
-        url = "https://download.qt.io/archive/qt/{0}/{1}/single/qt-everywhere-src-{1}" \
-            .format(self.version[:self.version.rfind('.')], self.version)
-        if tools.os_info.is_windows:
-            tools.get("%s.zip" % url, sha256='caab56aa411ef471d56bd6de54088bb38f0369de26dc874dd2703801d30d4ef9')
```

Comment:
I wasn't sure why this is done for windows, it works fine with the tar.xz and they are a lot smaller. But maybe there's a good reason for it and in that case we could just add another checksum in the config.

Reply:
IIRC There used to be a line ending difference between these packages, which made them compatible with only one platform. This may be not true any more.